src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -0,0 +1,208 @@
+package io.github.oliviercailloux.y2018.apartments.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.github.oliviercailloux.y2018.apartments.apartment.Apartment;
+
+import javax.json.bind.JsonbBuilder;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.ArrayList;
+import static com.google.common.base.Preconditions.checkArgument;
+
+import javax.json.bind.Jsonb;
+
+/**
+ * The Class JsonConvert contains all function to transform Apartment object to
+ * JSON and vice-versa.
+ * 
+ * @author Etienne CARTIER & Morgane FIOT
+ */
+public abstract class JsonConvert {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(JsonConvert.class);
+
+	/**
+	 * The method return the default path to JSON file.
+	 * 
+	 * @return <i>Path</i> where to read and write JSON.
+	 */
+	private static final Path apartmentPathJson() {
+		Path path = Paths.get("Apartment_Json.json");
```

Comment:
En faire une constante (variable final static), et envisager de la rendre publique.

Reply:
Fix in commit 26d5c5ac3cfd4a792018758a96b340395986b1cb